backends/xnnpack/partition/config/xnnpack_config.py

```diff
@@ -174,11 +175,19 @@ def _check_inputs_are_valid_dtypes(self, node, valid_dtypes):
             if arg_val.dtype not in valid_dtypes:
                 return False
 
+            # Check for mixed dtypes
```

Comment:
let's add a why log for when we fail. Let's try to add what node fails, and what the dtypes are of the inputs:

https://github.com/pytorch/executorch/blob/74e430812810840cc935d25fd64b558dd106afb6/backends/xnnpack/partition/config/generic_node_configs.py#L156-L159